CHANGELOG.rst

```diff
@@ -11,6 +11,12 @@ Change Log
 
 .. There should always be an "Unreleased" section for changes pending release.
 
+[5.15.0] - 2024-07-29
+---------------------
+Added
+~~~~~
+* Added ``start_trace`` to ``monitoring.utils`` so that manual spans in our monitoring backends can be created.
```

Comment:
We should probably use the public location of the addition in the changelog:

```suggestion
* Added ``start_trace`` to ``edx_django_utils.monitoring`` so that manual spans in our monitoring backends can be created.
```